Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -894,7 +894,7 @@ class ASFooterView: UICollectionReusableView {
     }
 
     var insets: CGFloat {
-        return UIScreen.main.bounds.size.width == self.frame.size.width && UIDevice.current.userInterfaceIdiom == .pad ? FirefoxHomeHeaderViewUX.Insets : FirefoxHomeUX.MinimumInsets
+        return UIScreen.main.bounds.size.width == self.frame.size.width && UIDevice.current.isPad ? FirefoxHomeHeaderViewUX.Insets : FirefoxHomeUX.MinimumInsets
```

Comment:
I suggest replacing this with a UITraitCollection based approach. You could be on an iPad, but in split screen mode.